test/integration/tests/tribalCouncil.ts

```diff
@@ -1,161 +0,0 @@
-import { Core, PodAdminGateway, PodFactory, RoleBastion } from '@custom-types/contracts';
-import { NamedAddresses, NamedContracts } from '@custom-types/types';
```

Comment:
Out of curiosity, did you not need to reduce the number of concurrent CI workers/parallelism when you removed this test? 

I've found before that when there are more workers than e2e tests, then they kind of "wrap around" and the spare worker starts to run the whole e2e test suite individually, taking forever

Reply:
Perhaps that's why the tests took so long. My plan was to adjust CI after these three cleanup prs get merged in.